### PR TITLE
Adding My Page

### PR DESCRIPTION
**Pull request by paulnorthpole**

### CODE REVIEW

File: _pages/PaulNorthPole.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: PaulNorthPole
+pageUrl: PaulNorthPole.github.io
+timestamp: 2017-04-26
```
Comment (brianamarie) [inline _pages/PaulNorthPole.yaml]: Hi @paulnorthpole! For the tests to pass, add another line at the end of the file with `---`, just like line 1. Let me know if you want or need any help! 😄

Comment (brianamarie) [inline _pages/PaulNorthPole.yaml]: Also, our tests look to see that the URL on line 3 is valid. Make sure that's a valid URL 👍

**Comment by paulnorthpole** [comment]
Thank you for your help @brianamarie,  so now i have to updated files on this request and its still checking the first file and failing.   I think I need to just somehow undo the merge request and start over.   Is there a way to do that?  if so, how?

**Comment by brianamarie** [comment]
@paulnorthpole First, I made one more change to remove the `http:` because our tests are picky 😉 . Hopefully this will get the tests to pass, and you don't need to start over.

However, IF you wanted to, you could click 'close' pull request, and either delete this branch or start with a new branch off of `master`. Does that answer your question? (PS I'll stick around long enough to make sure this tests goes green, because there is no reason it shouldn't!) 😄

**Comment by paulnorthpole** [comment]
Thanks @brianamarie.  Looks like it has passed now.   thanks for the help.

**Pull request merged**